rootfs/etc/profile.d/inputrc.sh

```diff
@@ -0,0 +1,7 @@
+# If we can, and do not have one, copy inputrc from localhost.
+[ -f ~/.inputrc ] && [ -f /localhost/.inputrc ] || {
```

Comment:
Don't you mean this?

```suggestion
[ ! -f ~/.inputrc ] && [ -f /localhost/.inputrc ] || {
```

Reply:
Yes.  Thanks!